packages/utils/src/audit-diff-finder.js

```diff
@@ -668,7 +669,9 @@ function findAuditDiffs(baseAudit, compareAudit, options = {}) {
   // If the only diff found was a numericValue/displayValue diff *AND* it seems like the result was flaky, skip it.
   // The result is likely flaky if the audit passed *OR* it was supposed to have details but no details items changed.
   const isAllPassing = compareAudit.score === 1 && baseAudit.score === 1;
+  const group = getGroupForAuditId(auditId);
   if (
+    group !== 'metrics' && // if metrics group audit is found, don't skip it
```

Comment:
this changes the behavior that was intentional if both were passing, is there a specific case you have in mind for changing this?

Reply:
I think @patrickhulce we agreed [here](https://github.com/GoogleChrome/lighthouse-ci/issues/689#issuecomment-948556023) on skipping this check when the group id is `metrics`.

Reply:
Ah thanks for the reminder it had been awhile :) sg!